models/volunteering-request-offer-base.js

```diff
@@ -35,8 +33,7 @@ const VolunteeringRequestOfferBaseSchema = new mongoose.Schema({
     default: [],
   },
   createdByUserId: {
-    type: Schema.Types.ObjectId,
```

Comment:
I truly think that using auth0 solely will cause headaches up the road. I find it better if we were using auth0 for authentication only and store users's meta-data in our own DB. To my believe using our own DB for meta-data will turn to be much more flexible. Tough as you wish.